frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,471 @@
+                        disabled
+                        :value="
+                          format(fromDate, 'dd/MM/yyyy') +
+                          ' - ' +
+                          format(toDate, 'dd/MM/yyyy')
+                        "
+                      />
+                    </div>
+                    <div class="w-full h-[1/3] relative">
+                      <SvgStripes class="absolute w-[6.9rem] right-0 z-0" />
+                      <div
+                        class="absolute ml-10 rounded-full border-input-placeholder text-input-placeholder w-16 h-16 self-center border-dashed border-2 right-2 bottom-2 flex text-center justify-center items-center text-xs pl-1.5 pr-1.5 dark:border-white dark:text-white"
+                      >
+                        <T keyName="journey.turn" />
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div
+                  class="h-[90%] w-0 rounded-b-r-3xl border-border-gray border-r-2 border-dashed"
+                ></div>
+              </div>
+            </div>
+            <div
+              class="text-text absolute inset-0 h-full w-full rounded-xl bg-white text-center [transform:rotateX(180deg)] [backface-visibility:hidden] dark:bg-background-dark"
+            >
+              <div
+                class="h-10 bg-border border-x-2 border-t-2 border-border-darker rounded-t-2xl flex items-center relative dark:border-border-blue-dark dark:bg-ticket-top-dark-bg"
+              >
+                <div
+                  class="absolute ml-5 rounded-full w-7 h-7 bg-border-gray inline-block self-center"
+                ></div>
+                <p class="ml-16 text-white font-bold text-xl">JourneyPlanner</p>
+                <div class="w-full flex justify-end h-full items-center">
+                  <SvgAirplaneIcon class="w-7 mr-5" />
+                </div>
+              </div>
+              <div class="flex h-5/6">
+                <div
+                  class="h-full w-full rounded-b-2xl bg-background border-border-gray border-x-2 border-b-2 flex justify-center dark:bg-border-dark dark:border-form-input-dark"
+                >
+                  <div class="h-full w-full flex flex-col items-end relative">
+                    <img
+                      class="absolute right-[50%] top-[25%] z-20 translate-x-[50%] -translate-y-[25%] w-2/5"
+                      :src="qrcode"
+                      alt="QR Code"
+                    />
+                    <div
+                      class="absolute ml-10 rounded-full border-input-placeholder text-input-placeholder w-16 h-16 self-center border-dashed border-2 right-2 bottom-2 flex text-center justify-center text-xs pl-1.5 pr-1.5 pt-1 z-40 dark:border-white dark:text-white"
```

Comment:
Tap to turn ist immer noch nicht in der Mitte wenn man die Karte umdreht
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/7debbc56-c6ce-41c7-9683-6e0193c13014)

Reply:
auch das passt bei mir